Sources/MapboxCoreNavigation/PassiveNavigationFeedbackType.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+
+/**
+ Feedback type is used to specify the type of feedback being recorded with `NavigationEventsManager.sendPassiveNavigationFeedback(_:type:description:)`.
+ */
+public enum PassiveNavigationFeedbackType: CustomStringConvertible {
+    
+    /// Indicates an incorrect visual.
+    case incorrectVisual(subtype: PassiveNavigationIncorrectVisualSubtype?)
+    
+    /// Indicates a road closure was observed.
+    case roadIssue(subtype: PassiveNavigationRoadIssueSubtype?)
+    
+    /// Indicates a wrong traffic.
+    case wrongTraffic(subtype: PassiveNavigationWrongTrafficSubtype?)
+    
+    /// The user’s location is not matching the real position of the vehicle.
+    case badGPS
```

Comment:
`positioningIssue`?

Reply:
There was a proposal to use "bad GPS", because "positioning issue" sounds like this is a problem of our SDK.